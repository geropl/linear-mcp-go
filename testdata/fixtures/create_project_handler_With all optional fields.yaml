---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 510
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation ProjectCreate($input: ProjectCreateInput!) {\n\t\t\tprojectCreate(input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tproject {\n\t\t\t\t\tid\n\t\t\t\t\tname\n\t\t\t\t\tdescription\n\t\t\t\t\tslugId\n\t\t\t\t\tstate\n\t\t\t\t\turl\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"input":{"name":"Test Project 2","teamIds":["234c5451-a839-4c8f-98d9-da00973f1060"],"description":"Test Description","leadId":"cc24eee4-9edc-4bfe-b91b-fedde125ba85","startDate":"2024-01-01","targetDate":"2024-12-31"}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"projectCreate":{"success":true,"project":{"id":"3e0c7d12-cfc3-4d38-bdf8-4f83a1be5841","name":"Test Project 2","description":"Test Description","slugId":"624ff0e39d61","state":"backlog","url":"https://linear.app/linear-mcp-go-test/project/test-project-2-624ff0e39d61"}}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"119-zej8a1Lf6SVe8494yLx5pWBKixA"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
