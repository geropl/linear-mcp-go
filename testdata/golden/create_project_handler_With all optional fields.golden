err: ""
output: |
    Project: Test Project 2
      ID: 3e0c7d12-cfc3-4d38-bdf8-4f83a1be5841
      State: backlog
      URL: https://linear.app/linear-mcp-go-test/project/test-project-2-624ff0e39d61
      Description: Test Description
